src/cli/job.js

```diff
@@ -0,0 +1,134 @@
+import { close, open } from 'testcafe-browser-tools';
+import { spawn } from 'child_process';
+
+const MAX_JOB_RESTART_COUNT = 3;
+
+//Job
+export default class Job {
+    constructor (options, browser, testServer) {
+        this.options = options;
+
+        this.options.timeout *= 1e3;
+
+        this.id = browser.browserName;
```

Comment:
How are you going to find out an error correspond to browser instance. In other word you will have one-to-many relationship.  Besides, you can use now:
```js
{ ..., browserName: 'chrome_56.234234' },
{ ..., browserName: 'chrome_57.223423 }'
```
Or am I missing something?